app/Http/Controllers/MailboxesController.php

```diff
@@ -137,7 +137,7 @@ public function update($id)
 
         $user = auth()->user();
         $mailbox_user = $user->mailboxesWithSettings()->where('mailbox_id', $id)->first();
-        if (!$mailbox_user && $user->isAdmin() || $user->isITHead()) {
+        if (!$mailbox_user && $user->isAdmin() || !$mailbox_user &&$user->isITHead()) {
```

Comment:
Change to
if (!$mailbox_user && ($user->isAdmin() || $user->isITHead())) {

Reply:
It is done